<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>518</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>510</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">
#minimizeButton {
  image: url(:/img/res/minimize-button1.png);
}

#minimizeButton, #maximizeButton {
  background-color: transparent;
  border-radius: 0px;
}

#minimizeButton:hover, #maximizeButton:hover {
  border-style: none;
  background-color: #80c7c7c7;
}

#minimizeButton:pressed, #maximizeButton:pressed {
  border-style: none;
  background-color: #80808080;
}

#closeButton {
  image: url(:/img/res/close-button1.png);
  background-color: transparent;
  border-radius: 0px;
}

#closeButton:hover {
	image: url(:/img/res/close-button2.png);
  border-style: none;
  background-color: #e81123;
}

#closeButton:pressed {
  image: url(:/img/res/close-button2.png);
  border-style: none;
  background-color: #8c0a15;
}

#fileBtn{
font: 9t &quot;华康方圆体W7(P)&quot;;
border-radius: 10px;
border-color: rgb(0, 170, 255);
background-color: rgb(170, 255, 255);
}

#fileBtn:pressed {

	background-color: rgb(170, 255, 255);
}
#startBtn{
font: 10t &quot;华康方圆体W7(P)&quot;;
border-radius: 10px;
border-color: rgb(0, 170, 255);
background-color: rgb(170, 255, 255);
}
#startBtn:pressed {
	background-color: rgb(170, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="ico">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/img/res/dog.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>第一题:无向图</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>297</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="minimizeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Minimize</string>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maximizeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Maximize</string>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="iconSize">
        <size>
         <width>34</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>310</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
     <property name="title">
      <string>建无向图</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="maxNum">
          <property name="placeholderText">
           <string>请输入你想要生成图的最大容量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newMap">
          <property name="text">
           <string>新建图</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QTableWidget" name="pointView">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>下标</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>值</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
     <property name="title">
      <string>增加点</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="addPoint">
        <property name="placeholderText">
         <string>输入你要增加的点的值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPointBtn">
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
     <property name="title">
      <string>修改点</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,2,1">
      <item>
       <widget class="QLineEdit" name="changePointIndex">
        <property name="placeholderText">
         <string>要修改点的下标</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="changePoint">
        <property name="placeholderText">
         <string>修改的值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changePointBtn">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
     <property name="title">
      <string>增加边</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="addLineIndex1">
        <property name="placeholderText">
         <string>下标1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addLineIndex2">
        <property name="placeholderText">
         <string>下标2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addLine">
        <property name="placeholderText">
         <string>新的边所存的值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addLineBtn">
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2" rowspan="4">
    <widget class="QTableWidget" name="mapView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>下标１</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>下标２</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>值</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
     <property name="title">
      <string>修改边</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="changeLineIndex1">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>要修改边的下标1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="changeLineIndex2">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>要修改边的下标2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="changeLIne">
        <property name="placeholderText">
         <string>修改的值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeLineBtn">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>当前有0个点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fromIndex">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>请输入你要开始从哪个点开始遍历，请输入该点的下标</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DFS">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>深度优先</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BFS">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>广度优先</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTextBrowser" name="logOut">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;华文新魏&quot;;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimizeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
